src/doc/trpl/concurrency.md

```diff
@@ -272,6 +258,8 @@ thread::spawn(move || {
     data[i] += 1;
 });
 #     }
+#
+#     thread::sleep_ms(50);
```

Comment:
That block is supposed to be a mirror of the block above it, but with most lines commented out to focus on the inner function body.

For whatever reason, it didn't include the sleep, even in the current version of the document (https://github.com/rust-lang/rust/blob/80def6c2447d23a624e611417f24cf0ab2a5a676/src/doc/trpl/concurrency.md)

This is effectively a no-op, since the commented parts get elided when rendered (see http://doc.rust-lang.org/book/concurrency.html), but it was clearly intended to be a mirror of the previous block at some point. This brings it back into line with that.